loginas/views.py

```diff
@@ -82,4 +83,9 @@ def user_logout(request):
     """
     restore_original_login(request)
 
-    return redirect(la_settings.LOGOUT_REDIRECT)
+    try:
+        url = request.META.get("HTTP_REFERER", "/").split('?next=')[1]
```

Comment:
There's an issue here, when the Referer has not been specified, this will go to `/` instead of `LOGOUT_REDIRECT`.

Reply:
Oh, no, I read it wrong, sorry.